### PR TITLE
PLATUI-1487: Update GETTING-STARTED.md with guidance for Looping

### PR DESCRIPTION
**Pull request by ejayaraman**
- Added a section about using Gatling's `Exec` method

### CODE REVIEW

**Review by vivrichards600 — APPROVED** [review]
Additions to the readme look good.

**Review by vivrichards600 — APPROVED** [review]
Link now appears to work as expected.

File: GETTING-STARTED.md [hunk 1]
```diff
@@ -252,7 +255,45 @@ The available placeholders are:<br>
 from 1 again when it reaches the max value. For example ${range-3} will be replaced with '001' the first time, '002' 
 the next and so on.
 
-### Creating a custom user feeder
+### Using Gatling's Exec method with performance-test-runner
+Gatling's Exec method is used to execute an action. Actions are usually requests like an HTTP request or editing a Gatling session.
```
Comment (JoPintoPaul) [inline GETTING-STARTED.md:259]:
```suggestion
Gatling's Exec method is used to execute an action. Actions are usually requests like an HTTP request. Additionally, actions can edit or debug a Gatling session.
```

File: GETTING-STARTED.md [hunk 2]
```diff
@@ -321,6 +362,46 @@ val setRandomTestId: List[ActionBuilder] = {
...
+#### Iterate asLongAs
+Use `asLongAs` to iterate [as long as](https://gatling.io/docs/gatling/reference/3.4/general/scenario/#aslongas) the 
+condition is satisfied and the duration has not been reached.
+
+For example, to repeat a request `asLongAs` the status of a page is **not** `200`:
+
+```scala
+def getTurnoverPage: List[ActionBuilder] = {
+  asLongAs(session => 
+    !session.attributes.get("turnOverPageStatus").contains(200)) {
```
Comment (JoPintoPaul) [inline GETTING-STARTED.md:389]: Something about this code example looks a little strange, is there a link somewhere to the Gatling API for Scala that I could look at?

Comment (ejayaraman) [reply GETTING-STARTED.md:389]: Hi, @JoPintoPaul. This is where `asLongAs` is defined: https://github.com/gatling/gatling/blob/v3.4.2/gatling-core/src/main/scala/io/gatling/core/structure/Loops.scala#L75

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**